src/locales/en.json

```diff
@@ -140,8 +138,8 @@
   "Timezone": "Timezone",
   "Time zone updated successfully": "Time zone updated successfully",
   "To close the purchase order, select all.": "To close the purchase order, select all.",
-  "Unable to update barcode identification preference.": "Unable to update barcode identification preference.",
-  "Unable to update force scan preference.": "Unable to update force scan preference.",
+  "Unable to update barcode identification preference since no product store config.": "Unable to update barcode identification preference since no product store config.",
```

Comment:
Unable to update barcode identification preference since no product store config found.

Reply:
Same for the below message as well.